translate/google/cloud/translate_v2/_http.py

```diff
@@ -29,15 +29,17 @@ class Connection(_http.JSONConnection):
     :param client_info: (Optional) instance used to generate user agent.
     """
 
-    def __init__(self, client, client_info=None):
+    DEFAULT_API_ENDPOINT = "https://translation.googleapis.com"
+
+    def __init__(self, client, client_info=None, api_endpoint=None):
```

Comment:
Simpler to default to the class attr, e.g.:
```suggestion
    def __init__(self, client, client_info=None, api_endpoint=DEFAULT_API_ENDPOINT):
```

which lets you remove the `if` below.